src/Service/Context/PageContext.php

```diff
@@ -59,6 +59,13 @@ class PageContext {
   ];
 
   /**
+   * JavaScript path.
+   *
+   * @var array
+   */
+  private $jsPath;
```

Comment:
The reason why $jsPath is private, or in a broader sense, why in PageContext almost all class variables and functions are private, is because there is no need for any of those privates to be protected or public. i.e. OOP "minimum visibility" guideline.